style.css

```diff
@@ -1,69 +1,66 @@
-
-body{
-	background-color: black;
-	color: white;
-	margin: 0 auto;
+body {
+  background-color: black;
+  color: white;
+  margin: 0 auto;
 }
 
 .fluid-container {
-	margin: 30px;
-	text-align: center;
+  margin: 30px;
+  text-align: center;
 }
 
-.title{
-	margin: 0;
-}
-#header-img{
-	width: 100%;
-	height: 900px;
+.title {
+  margin: 0;
 }
 
-.quote{
-	margin: 50px;
-	padding: 30px;
-	text-align: center;
-	font-size: 20px;
-	font-style: italic;
+#header-img {
+  width: 100%;
+  height: 900px;
 }
 
-#results{
-	display: grid;
-	grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
-	margin: 40px;
+.quote {
+  margin: 50px;
+  padding: 30px;
+  text-align: center;
+  font-size: 20px;
+  font-style: italic;
+}
 
+#results {
+  display: grid;
+  grid-template-columns: repeat(auto-fill, minmax(400px, 1fr));
+  margin: 40px;
 }
 
 #img-id:hover {
-    transform: scale(1.2);
-    transition: transform .2s;
-    border: 2px solid white;
+  transform: scale(1.2);
+  transition: transform 0.2s;
+  border: 2px solid white;
 }
 
-img{
-	display: inline-block;
-	height: 300px;
-	width: 400px;
-	margin: 5px;
+img {
+  display: inline-block;
+  height: 300px;
+  width: 400px;
+  margin: 5px;
 }
 
-#ytplayer{
-	padding-left: 50px;
-	padding-top: 50px;
-	width: 100%;
-	margin: 0 auto;
+#ytplayer {
+  padding-left: 50px;
+  padding-top: 50px;
+  width: 100%;
+  margin: 0 auto;
+  opacity: 1 !important;
```

Comment:
`!important` isn't my favorite solution to override a CSS declaration, but moving it to the bottom of the stylesheet doesn't fix it, which tells me that it might be a Youtube/Chrome thing that's affecting this selector.